packages/common/src/sites/_internal/_ensure-event-list-card.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  IModel,
+  IDraft,
+  getProp,
+  cloneObject,
+  deleteProp,
+  setProp,
+} from "../..";
```

Comment:
please import from files - these type of paths cause issues with the node resolver (sometimes!)